tests/integration/test_logcollector/test_age/test_age_basic.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import sys
+
+import pytest
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, AGENT_DETECTOR_PREFIX
+from wazuh_testing.tools.time import time_to_seconds
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_age.yaml')
+
+WINDOWS_FOLDER_PATH = r'C:\testing_age' + '\\'
```

Comment:
I recommend you to change this directory to `%temppdir`. This is the temporary directory for Windows. You can use the function `gettempdir()` to detect this directory independently of the OS.

Reply:
Done 41fd7460dc64d554987256b75777f84190bbdb4c and a6da7de49152ea31d69d3cf25dcb3b82e288dc95